Automattic/sensei
# Remove HPPS migration tool

Resolves #7311

## Proposed Changes
* Remove the HPPS migration tool.

## Testing Instructions

1. Go to Sensei LMS > Tools. Make sure you don't see the tool.
2. Go to Sensei LMS > Settings > Experimental Features, enable HPPS. 
3. Go to Sensei LMS > Tools. Make sure you don't see the tool.
4. Go to Sensei LMS > Settings > Experimental Features, enable sync.
5. Go to Sensei LMS > Tools. Make sure you don't see the tool.

## Pre-Merge Checklist
